Userland/Libraries/LibWeb/WebVTT/VTTCue.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2024, Jamie Mansfield <jmansfield@cadixdev.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWeb/Bindings/Intrinsics.h>
+#include <LibWeb/Bindings/VTTCuePrototype.h>
```

Comment:
Based on the errors I'm seeing on MacOS CI, I *think* you might need to move this include to `VTTCue.h` since you use some IDL-defined enums there.

I'm not sure though, since I can't reproduce the issue locally on Mac or Linux :shrug:

Reply:
I can't reproduce lovely either, but I had similar issues lovely with `VTTRegion`. I'll move this shortly :)